src/textual/_compositor.py

```diff
@@ -45,12 +45,23 @@ class ReflowResult(NamedTuple):
 class MapGeometry(NamedTuple):
     """Defines the absolute location of a Widget."""
 
-    region: Region  # The (screen) region occupied by the widget
-    order: tuple[tuple[int, ...], ...]  # A tuple of ints defining the painting order
-    clip: Region  # A region to clip the widget by (if a Widget is within a container)
-    virtual_size: Size  # The virtual size  (scrollable region) of a widget if it is a container
-    container_size: Size  # The container size (area not occupied by scrollbars)
-    virtual_region: Region  # The region relative to the container (but not necessarily visible)
+    region: Region
+    """The (screen) region occupied by the widget."""
+    order: tuple[tuple[int, int, int], ...]
```

Comment:
Is this correct? I have a feeling the outer tuple can contain both tuples and ints.

Reply:
That _became_ correct. The commit that introduced that change also tweaked a couple of other lines of code to make sure that was correct and it includes a commit message that explains why that is a good fix. (From my point of view.)